src/components/FSHControls.js

```diff
@@ -167,6 +213,49 @@ export default function FSHControls(props) {
     if (version === '' && config.version) setVersion(config.version);
   }
 
+  async function fetchExampleFSH(event, value) {
+    if (!value.endsWith('.fsh')) return;
+    const exampleMetadata = props.exampleMetadata[value];
+    setIsFetchingExample(true);
+    setCurrentExampleName(exampleMetadata.name);
+    const utf8Decoder = new TextDecoder('utf-8');
+    let responseReader = await fetch(exampleMetadata.path).then((response) => response.body.getReader());
+    let fshString = '';
+    const { value: chunk } = await responseReader.read();
+    fshString += utf8Decoder.decode(chunk);
+    setIsFetchingExample(false);
+    setCurrentExample(fshString);
+  }
+
+  function updateExampleValue(text) {
+    setCurrentExample(text);
+  }
+
+  function handleCopyToClipboard() {
+    navigator.clipboard.writeText(currentExample);
+  }
+
+  const renderItem = (node) => (
+    <Tooltip
+      key={node.id}
+      title={
+        props.exampleMetadata[node.id] && props.exampleMetadata[node.id].description
+          ? props.exampleMetadata[node.id].description
+          : node.name
+      }
+      placement="left"
```

Comment:
I agree that the tooltips are a bit far away. I have them set to appear on the left of the menu, but I guess when the window is too small they appear on the right instead. I'll change this so that they appear on the bottom!

Reply:
Oh yikes I didn't realize my screen was so small! That did explain my confusion when I saw this line has "left" and I was seeing it on the right 😂 

I did try moving the window to be the full width of my laptop screen and they still showed up on the right for me. We definitely don't need to support crazy small screen widths, but I think laptop-sized (very official) is reasonable to support. Maybe bottom is the better bet, since I'm not sure how to control when they are rearranged.

Reply:
If they're not showing up on the left side on a 16-inch laptop screen then its fair to say it should be changed 😂. 

I now have them appearing on the bottom right to avoid covering up the rest of the file tree as much as possible. Let me know how that looks to you! They can still appear somewhat off to the side for some of the menu items, but its definitely an improvement. If you still think that's a bit too far away, I can change the tooltips to appear centered at the bottom.

Reply:
Technically I have a measly 15-inch laptop 😂 

The one thing that still bugs me a little bit is that when the description is long, the little tooltip arrow ends up in the middle of the tooltip box like this:

![image](https://user-images.githubusercontent.com/30803904/120719475-58463000-c498-11eb-8c3b-386657b3b36d.png)

But when the description is short, it ends up at the left side of the tooltip like this:

![image](https://user-images.githubusercontent.com/30803904/120719508-66944c00-c498-11eb-821a-ce82c9667bca.png)

That might just be something that bothers me, so if that doesn't bother you, I think this is okay! I might just not be understanding tooltip logic correctly.

Reply:
I have the tooltips set to appear on the bottom right to avoid covering up other items in the tree, but I guess when the description gets too the tooltips are just centered instead. It would be better for consistency's sake to just have it appear centered at the bottom the entire time, especially since I imagine most descriptions will be kind of lengthy. I'll change that!

Reply:
Addressed in `5e25d7a` which centered all of the tooltips, let me know how it looks!

Reply:
It looks good to me! I like the consistency. I still agree that left seems the most ideal, but since that doesn't stay consistent, I like this approach!